README.md

```diff
@@ -2,111 +2,78 @@
-| `s3.max_dimension` | (none) | If set, a resized copy will be created for any image whose dimensions exceed this number. This copy will be served in place of the original.
-| `phash_distance` | 0 | Determines what binary distance is considered a match when querying with a phash fingeprint. Using more than 8 is not recommended and may lead to large amounts of false positives. **Note**: The [pg-spgist_hamming extension](#phash-distance-matching) must be installed to use distance matching, otherwise you will get errors. |
-| `favicon_path` | (none) | Location where favicons for linked sites should be stored. Leave empty to disable. |
-| `draft_time_limit` | (24h) | Time, in seconds, before a draft is deleted. |
-| `profiler_port` | 0 | Port on which to serve pprof output. Omit to disable entirely. |
-| `postgres.max_open_conns` | (0) | Maximum number of concurrent open connections to the database. |
-| `postgres.max_idle_conns` | (0) | Maximum number of concurrent idle database connections. |
-| `postgres.conn_max_lifetime` | (0) | Maximum lifetime in minutes before a connection is released. |
+2. API key authentication: To use an API key, set the `ApiKey` header to the user's API key value.
 
 ## SSL (HTTPS)
 
-Stash-box supports HTTPS with some additional work.  First you must generate a SSL certificate and key combo.  Here is an example using openssl:
+Stash-box is runnable, preferably over HTTPS, for added security, but it requires some setup. You'll need to generate an SSL certificate and key pair to set this up. Or use a TLS terminating proxy of your choice, such as Traefik, Nginx (unsupported), or Caddy Server (unsupported)
+
+Here's an example of how you can do this using OpenSSL:
 
 `openssl req -x509 -newkey rsa:4096 -sha256 -days 7300 -nodes -keyout stash-box.key -out stash-box.crt -extensions san -config <(echo "[req]"; echo distinguished_name=req; echo "[san]"; echo subjectAltName=DNS:stash-box.server,IP:127.0.0.1) -subj /CN=stash-box.server`
 
-This command would need customizing for your environment.  [This link](https://stackoverflow.com/questions/10175812/how-to-create-a-self-signed-certificate-with-openssl) might be useful.
 
-Once you have a certificate and key file name them `stash-box.crt` and `stash-box.key` and place them in the directory where stash-box is run from. Stash-box detects these and starts up using HTTPS rather than HTTP.
+You might need to modify the command for your specific setup. You can find more information about creating a self-signed certificate with OpenSSL [here](https://stackoverflow.com/questions/10175812/how-to-create-a-self-signed-certificate-with-openssl).
+
+Once you've generated the certificate and key pair, make sure they're named `stash-box.crt` and `stash-box.key` respectively, and place them in the same directory as stash-box. When Stash-box detects these files, it will use HTTPS instead of HTTP.
 
 ## PHash Distance Matching
```

Comment:
Would change it to `## pHash distance matching` to keep capitalization consistent with other headings.

Reply:
Resolved in next commit.